### PR TITLE
Cleaned up addressable loading in SynthDetScenario

### PR DESCRIPTION
**Pull request by sleal-unity**
This PR cleans up addressable loading in SynthDetScenario by using built in plural addressable load functions instead of manually loading individual assets.

### CODE REVIEW

File: SynthDet/Assets/Scripts/Runtime/SynthDetScenario.cs [hunk 1]
```diff
@@ -19,142 +20,102 @@ namespace SynthDet.Scenarios
...
+                    Quit();
+                return false;
             }
         }
 
         protected override void OnAwake()
         {
             base.OnAwake();
 
-            Caching.ClearCache();
```
Comment (JonathanHUnity) [inline SynthDet/Assets/Scripts/Runtime/SynthDetScenario.cs:89]: Could we keep this line please? The cache has led to some nasty debugging issues where we thought we were doing things right but in fact were not.

File: SynthDet/Assets/Scripts/Runtime/SynthDetScenario.cs [hunk 2]
```diff
@@ -19,142 +20,102 @@ namespace SynthDet.Scenarios
...
-                    }
-                    case AssetLoadingStatus.Failed:
-                    {
-                        Quit();
-                        return false;
-                    }
-                    default:
-                        return false;
-                }
+                if (m_LoadingStatus == AssetLoadingStatus.Complete)
```
Comment (JonathanHUnity) [inline SynthDet/Assets/Scripts/Runtime/SynthDetScenario.cs:34]: Out of curiosity, why can't we completely block the main thread until all of the loading is complete?

**Review by JonathanHUnity — APPROVED** [review]
lgtm. Couple points here to look at.

File: SynthDet/Assets/Scripts/Runtime/SynthDetScenario.cs (hunk 1 above)
Comment (sleal-unity) [reply SynthDet/Assets/Scripts/Runtime/SynthDetScenario.cs:89]: Sure thing

File: SynthDet/Assets/Scripts/Runtime/SynthDetScenario.cs (hunk 2 above)
Comment (sleal-unity) [reply SynthDet/Assets/Scripts/Runtime/SynthDetScenario.cs:34]: I think I ran into some problems with waiting on the async Task returned from addressables methods, but I'll experiment with this again to see if we can just block the main thread like you suggested.

Comment (sleal-unity) [reply SynthDet/Assets/Scripts/Runtime/SynthDetScenario.cs]: Alrighty, the whole loading process should block the main thread now.

**Comment by sleal-unity** [comment]
Just wanna call out that I'll merge this code in once I get this PR finalized over on the perception side:
https://github.com/Unity-Technologies/com.unity.perception/pull/281

**Pull request merged**